src/i18n/utils.ts

```diff
@@ -11,6 +11,17 @@ export function useTranslations(lang: keyof typeof ui) {
 	};
 }
 
+export function isOutdated(lang: keyof typeof ui) {
+	if ("version" in ui[lang]) {
+		return (
+			(ui[lang] as (typeof ui)[typeof lang])["version"] <
+			ui[defaultLang]["version"]
+		);
+	} else {
+		return true;
+	}
```

Comment:
All translations should contain the same keys....

```suggestion
	return (
		(ui[lang] as (typeof ui)[typeof lang])["version"] <
		ui[defaultLang]["version"]
	);
```

Reply:
This confuses me - while I can see where you are coming from, the [current `useTranslations` function](https://github.com/auxolotl/website/blob/49b5347903056007e65dee35a91edf2ab6b8cac5/src/i18n/utils.ts#L10) contains an `ui[lang] || ui[defaultLang]` statement, which would be redundant if all translations contain the same keys.